content/Lab2/20_Testing/1_test_setup.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Test Setup"
+weight: 1
+---
+
+## Test Setup
+
+To test this setup we will deploy 2 linux machines. One will act as a bastion server to which the user will SSH to. The other machine will be the test machine (without direct internet connection) from which the test traffic to the internet will be generated.
+
+### **1. Deploy the Bastion** 
+
+1. Navigate to the EC2 Dashboard: Click on the "Services" dropdown menu, select "EC2" under the "Compute" section.
+2. Click on the "Launch Instance" button.
+3. choose an Ubuntu AMI.
+4. In instance type, choose "t2.micro".
+5. Select the key pair created earlier. This will be used to SSH into your instance.
+6. Select Spoke VPC and one Spoke subnet (eg Spoke1) for deployment.
+7. Enable "Auto-assign Public IP" so that the instance will have a public IP.
+8. Create a new security group or use an existing one. Open the required ports for SSH,HTTP and any other services you want to use.
+9. Click on the "Launch" button.
+
+![bastion](../../images/bastion.png)
+
+### **2. Deploy the Test machine** 
+1. Navigate to the EC2 Dashboard: Click on the "Services" dropdown menu, select "EC2" under the "Compute" section.
+2. Click on the "Launch Instance" button.
+3. choose an Ubuntu AMI.
+4. In instance type, choose "t2.micro".
+5. Select the key pair created earlier. This will be used to SSH into your instance.
+6. Select Spoke VPC and one Spoke subnet (eg Spoke2) for deployment. 
+>Note: Use a different subnet for your test machine than what is used for bastion.
+7. Disable "Auto-assign Public IP"
+8. Create a new security group or use an existing one. Open the required ports for SSH,HTTP and any other services you want to use.
+9.  Click on the "Launch" button.
+
+![testmc](../../images/test-mc.png)
```

Comment:
These images were not visible in preview. Please ensure that they are referenced correctly